### PR TITLE
Fixed #34917 . Links in the Django admin should be underlined.

### PR DESCRIPTION
**Pull request by Myrausman**
Fixed ticket [#34917](https://code.djangoproject.com/ticket/34917) - Links in the Django admin are now underlined.

Added CSS rule for underlining links in Django admin. 
for links i have added ` text-decoration: underline;`

### CODE REVIEW

File: django/contrib/admin/static/admin/css/base.css [hunk 1]
```diff
@@ -105,16 +105,18 @@ body {
...
 }
 
 a:focus, a:hover {
     color: var(--link-hover-color);
+    text-decoration: underline;
 }
 
 a:focus {
     text-decoration: underline;
+    text-decoration: underline;
```
Comment (knyghty) [inline django/contrib/admin/static/admin/css/base.css:119]: This is the same as the line above.

**Review by knyghty — COMMENTED** [review]
Hi @Myrausman,

Thanks for this. In the ticket, I suggested that we should only underline the links in the main part of the page (not the sidebar or header, other than the links in the header that already underlined). Could you update your work here?

**Comment by Myrausman** [comment]
```

/* LINKS */

a:link, a:visited {
    color: var(--link-fg);
    transition: color 0.15s, background 0.15s;
}

a:focus, a:hover {
    color: var(--link-hover-color);
}

a:focus {
    text-decoration: underline;
}
a img {
	@@ -123,7 +125,7 @@ a img {

a.section:link, a.section:visited {
    color: var(--header-link-color);
    text-decoration: none;
}
a.section:focus, a.section:hover {
    text-decoration: underline;
}

#content-main a:link,
#content-main a:visited {
    text-decoration: underline;
}


```
My apologies for not understanding clearly before. Can you confirm if this is okay, or should I make any other changes?

**Comment by knyghty** [comment]
I suggest you try some things out and check locally if the right links are underlined and then update your code here.

**Comment by Myrausman** [comment]
I believe there are some errors in this branch, including commits attributed to me that I did not make. Therefore, I've decided to create a new pull request.

**Pull request closed without merging**